packages/video_player/video_player_platform_interface/lib/messages.dart

```diff
@@ -156,7 +156,6 @@ class VideoPlayerApi {
       throw PlatformException(
         code: 'channel-error',
         message: 'Unable to establish connection on channel.',
-        details: null,
```

Comment:
This is an auto-generated file that should not be hand-edited (per the comment at the top).

The right fix here would be to change the Pigeon generation to `messages.g.dart` as we've been using in newer plugins (at the bottom of pigeons/messages.dart) and renaming this file accordingly.

Reply:
fixed in [17dce6f](https://github.com/flutter/plugins/pull/6163/commits/17dce6fd7208e329c6f5df7032d91c7501ae6872)